### ISSUE TITLE
Publish output isn't as pretty as it could be

### ISSUE THREAD

**Issue by isaacs** [npm/npm#8761]
```
$ npm pub
▀ ╢░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░╟
```

I <3 the progress bar and the spinner.

But would it be ok to have it clear the line and do the `+ package@1.2.3` line like it used to?  Makes it clearer what actually ended up published.

**Comment by iarna** [comment]
It probably is printing that out and then stomping on it with the prograss bar. =p

Yeah, that's not intended =D

**Comment by iarna** [comment]
This was fixed in 0848698e! Yay!
